app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -5,6 +5,16 @@
 //!
 //! To learn more about component groups, see the [Component Browser Design
 //! Document](https://github.com/enso-org/design/blob/e6cffec2dd6d16688164f04a4ef0d9dff998c3e7/epics/component-browser/design.md).
+//!
+//! # Header and its Shadow
```

Comment:
Why "shadow" is written in upper case?

Reply:
It's a [style of academic writings](https://www.scribbr.com/academic-writing/capitalization-titles-headings/) which I'm used to for some reason. Made it lowercase.